package.json

```diff
@@ -75,6 +76,7 @@
     "scroll-to": "0.0.2",
     "semver": "^5.1.0",
     "weex-vue-framework": "2.0.5-weex.1",
+    "weex-rx-framework": "0.1.13",
```

Comment:
It's better to separate the commit with two parts. 1 is adding frameworks config script. 2 is adding rx dep. And the second one is recommended to go to `jsfm-feature-0.18` first. We can have a face-to-face talk later. Don't mind about this :-)